src/lib/y2security/selinux_config.rb

```diff
@@ -139,12 +140,34 @@ def configurable?
       Yast::ProductFeatures.GetBooleanFeature("globals", "selinux_configurable")
     end
 
+    def needed_packages
+      pattern = Yast::OSRelease.ReleaseName =~ /micro/i ? MICRO_OS_PATTERN : PATTERN
+
+      [pattern, TOOLS_PACKAGE, POLICY_PACKAGE]
+    end
+
```

Comment:
Not sure if this is the right way to proceed or it would be better asking to the package system or even other way. What do you think?